### DISCUSSION TITLE
Be notified when pictures are added in album

### DISCUSSION THREAD

**Discussion by axgd-code** [immich-app/immich#625] [General]
As the owner of an album, I would like to be notify when someone add pictures.
As a viewer of an album, I would like to be notify when someone add pictures.

- in the webapp, I think that a rss feed could be a simple solution. A webhook could be more interesting. 
- in the mobile app, receive a push notification could be very nice too !

**Comment by panoti** [comment]
Hi @llexaxell, can you create a feature issue in 'Issues' tab? It will be easier for us to track your requirement.

**Reply by axgd-code** [reply]
done : https://github.com/immich-app/immich/issues/627

**Reply by panoti** [reply]
Thank you
